core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2717,73 +2603,48 @@ public String toString() {
+          .setDescription("When a client reads from a remote worker, the maximum number of "
+              + "messages to buffer by the client.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
-  public static final PropertyKey USER_NETWORK_NETTY_CHANNEL_POOL_DISABLED =
-      new Builder(Name.USER_NETWORK_NETTY_CHANNEL_POOL_DISABLED)
-          .setDefaultValue(false)
-          .setDescription("Disable netty channel pool. This should be turned on if the client "
-              + "version is >= 1.3.0 but server version is <= 1.2.x.")
+  public static final PropertyKey USER_NETWORK_READER_CHUNK_SIZE_BYTES =
+      new Builder(Name.USER_NETWORK_READER_CHUNK_SIZE_BYTES)
+          .setDefaultValue("64KB")
+          .setDescription("When a client reads from a remote worker, the maximum chunk size.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
-  public static final PropertyKey USER_NETWORK_NETTY_WRITER_BUFFER_SIZE_PACKETS =
-      new Builder(Name.USER_NETWORK_NETTY_WRITER_BUFFER_SIZE_PACKETS)
+  public static final PropertyKey USER_NETWORK_WRITER_BUFFER_SIZE_MESSAGES =
+      new Builder(Name.USER_NETWORK_WRITER_BUFFER_SIZE_MESSAGES)
           .setDefaultValue(16)
-          .setDescription("When a client writes to a remote worker, the maximum number of packets "
+          .setDescription("When a client writes to a remote worker, the maximum number of messages "
               + "to buffer by the client.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
-  public static final PropertyKey USER_NETWORK_NETTY_WRITER_CLOSE_TIMEOUT_MS =
-      new Builder(Name.USER_NETWORK_NETTY_WRITER_CLOSE_TIMEOUT_MS)
-          .setAlias(new String[]{"alluxio.user.network.netty.writer.close.timeout.ms"})
-          .setDefaultValue("30min")
-          .setDescription("The timeout to close a netty writer client.")
+  public static final PropertyKey USER_NETWORK_WRITER_CHUNK_SIZE_BYTES =
+      new Builder(Name.USER_NETWORK_WRITER_CHUNK_SIZE_BYTES)
+          .setDefaultValue("64KB")
+          .setDescription("When a client writes to a remote worker, the maximum packet size.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
-  public static final PropertyKey USER_NETWORK_NETTY_WRITER_FLUSH_TIMEOUT =
-      new Builder(Name.USER_NETWORK_NETTY_WRITER_FLUSH_TIMEOUT)
+  public static final PropertyKey USER_NETWORK_WRITER_CLOSE_TIMEOUT_MS =
+      new Builder(Name.USER_NETWORK_WRITER_CLOSE_TIMEOUT_MS)
+          .setAlias(new String[]{"alluxio.user.network.writer.close.timeout.ms"})
           .setDefaultValue("30min")
-          .setDescription("The timeout to wait for flush to finish in a netty writer.")
+          .setDescription("The timeout to close a writer client.")
```

Comment:
how is this different from the other client timeouts earlier, like `USER_NETWORK_DATA_TIMEOUT_MS`?

Reply:
This is specifically used for closing the data writer.